tests/aws/services/stepfunctions/v2/activities/test_activities.py

```diff
@@ -0,0 +1,196 @@
+    ActivityTemplate,
+)
+from tests.aws.services.stepfunctions.utils import (
+    create_and_record_execution,
+)
+
+
+@markers.snapshot.skip_snapshot_verify(paths=["$..loggingConfiguration", "$..tracingConfiguration"])
+class TestActivities:
+    @markers.aws.validated
+    def test_activity_task(
+        self,
+        aws_client,
+        create_iam_role_for_sfn,
+        create_state_machine,
+        create_activity,
+        sfn_activity_consumer,
+        sfn_snapshot,
+    ):
+        activity_name = f"activity-{short_uid()}"
+        create_activity_output = create_activity(name=activity_name)
+        activity_arn = create_activity_output["activityArn"]
+        sfn_snapshot.add_transformer(RegexTransformer(activity_arn, "activity_arn"))
+        sfn_snapshot.add_transformer(RegexTransformer(activity_name, "activity_name"))
+        sfn_snapshot.match("create_activity_output", create_activity_output)
+
+        sfn_activity_consumer(
+            template=ActivityTemplate.load_sfn_template(ActivityTemplate.BASE_ID_ACTIVITY_CONSUMER),
+            activity_arn=activity_arn,
+        )
+
+        template = ActivityTemplate.load_sfn_template(ActivityTemplate.BASE_ACTIVITY_TASK)
+        template["States"]["ActivityTask"]["Resource"] = activity_arn
+        definition = json.dumps(template)
+
+        exec_input = json.dumps({"Value1": "Hello", "Value2": "World"})
+        create_and_record_execution(
+            aws_client.stepfunctions,
+            create_iam_role_for_sfn,
+            create_state_machine,
+            sfn_snapshot,
+            definition,
+            exec_input,
+        )
+
+    @markers.aws.validated
+    def test_activity_task_no_worker_name(
+        self,
+        aws_client,
+        create_iam_role_for_sfn,
+        create_state_machine,
+        create_activity,
+        sfn_activity_consumer,
+        sfn_snapshot,
+    ):
+        activity_name = f"activity-{short_uid()}"
+        create_activity_output = create_activity(name=activity_name)
+        activity_arn = create_activity_output["activityArn"]
+        sfn_snapshot.add_transformer(RegexTransformer(activity_arn, "activity_arn"))
+        sfn_snapshot.add_transformer(RegexTransformer(activity_name, "activity_name"))
+        sfn_snapshot.match("create_activity_output", create_activity_output)
+
+        template_consumer = ActivityTemplate.load_sfn_template(
+            ActivityTemplate.BASE_ID_ACTIVITY_CONSUMER
+        )
+        del template_consumer["States"]["GetActivityTask"]["Parameters"]["WorkerName"]
+        sfn_activity_consumer(template=template_consumer, activity_arn=activity_arn)
+
+        template = ActivityTemplate.load_sfn_template(ActivityTemplate.BASE_ACTIVITY_TASK)
+        template["States"]["ActivityTask"]["Resource"] = activity_arn
+        definition = json.dumps(template)
+
+        exec_input = json.dumps({"Value1": "Hello", "Value2": "World"})
```

Comment:
We have to submit some input to these state machines, and with the tests we want to follow how and where the input is reported in the logs. But I agree, it would be preferable to address this. I have changed this to a smaller input.